src/shared/components/guidedTour/steps/areasStep.component.tsx

```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { Tag } from '../components/tag/tag.component';
+import { TagsWrapper } from '../components/tag/tag.styles';
+import { setArea, setLevel } from '../../../../modules/filters/filters.actions';
+import { useContentfulData } from '../../../hooks/useContentfulData/useContentfulData';
+import { QuadrantPositions } from '../../radar/radar.types';
+import { highlightBlips, unHighlightBlips } from './utils';
+
+export const AreasStep = () => {
+  const [activeTag, setActiveTag] = useState('');
+  const { radarTechnologies, radarQuadrants } = useContentfulData();
+  const dispatch = useDispatch();
+
+  const defaultQuadrant = radarQuadrants[0];
+  const currentQuadrant = radarQuadrants.find((quadrant) => quadrant.name === activeTag);
+
+  const handleQuadrantClick = (value: string, position: QuadrantPositions) => {
+    if (activeTag === value) return;
+    unHighlightBlips(radarTechnologies);
+    const technologiesToHighlight = radarTechnologies.filter((tech) => tech.quadrant === position);
+    highlightBlips(technologiesToHighlight);
+    setActiveTag(value);
+    dispatch(setArea(value));
+  };
+
+  useEffect(() => {
+    setTimeout(() => {
```

Comment:
There is a weird bug in this library, causing it to break when `redux` is used. By using `setTimeout` with `0ms` we are pushing `dispatch` further in the call stack, which results in the library working properly and not causing a delay